test/fixtures/fizzbuzz/fizzbuzz.rb

```diff
@@ -1,10 +1,11 @@
+# Padding
 def fizzbuzz_convert(n)
   if n % 15 == 0
     "fizzbuzz"
-  elsif n % 3 == 0
-    "fizz"
   elsif n % 5 == 0
     "buzz"
+  elsif n % 3 == 0
+    "fizz"
```

Comment:
I changed the order of "fizz" and "buzz" since there are test cases I want to test that TTNT just selectes test related to "fizz". If "fizz" line is above the "buzz" line, it means "fizz" line is "covered" by the coverage of "buzz_test.rb". Thus any change to "fizz" line leads to select "buzz_test.rb"...